include/ShearStressTransportEquationSystem.h

```diff
@@ -79,6 +84,10 @@ class ShearStressTransportEquationSystem : public EquationSystem {
   std::vector<stk::mesh::Part *> wallBcPart_;
 
   bool resetTAMSAverages_;     
+
+  const double tkeMinValue_{1.0e-8};
+  const double sdrMinValue_{1.0e-8};
+  const double sdrMaxValue_{1.0e12};
```

Comment:
Sorry, I realize this was in the CDP snippet input file I copied, but it's not actually used in the CDP code anywhere.  I don't know if it really does anything here as I'm not sure SDR gets that big, but we might want to remove it.

Reply:
I removed the max clipping.